examples/fluent-bit/cloudwatchlogs-emf/README.md

```diff
@@ -77,9 +77,24 @@ And then setup a port mapping on the FireLens container:
               }],
 ```
 
-See the full task definition in the [emf-over-tcp](emf-over-tcp) directory.
+See the full task definition in the [emf-over-tcp](emf-over-tcp) directory. The setup steps here are again similar to using the [CW Agent to collect EMF in ECS](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Generation_CloudWatch_Agent.html).
 
-### AWSVPC or Host Mode
+#### Note on FLUENT_HOST hostname
+
+The above linked steps and the bridge network mode example task definition in this tutorial use a Docker link to create a hostname in the app container for your Fluent Bit/FireLens container. It should be noted that [FireLens will set a hostname in app containers for Fluent Bit](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html) called `FLUENT_HOST`. However, the AWS EMF libraries require an environment variable to reach the collection agent:
+
+```
+"environment": [
+			{
+                "name": "AWS_EMF_AGENT_ENDPOINT",
+                "value": "tcp://fluentbit:25888"
+            }
+			],
+```
+
+It is not possible to reference the value of the `FLUENT_HOST` environment variable in the value of this environment variable. However, if you are constructing EMF directly in your application code and sending it over TCP instead of using the AWS EMF libraries, using `FLUENT_HOST` may be a desirable option.
```

Comment:
Is FLUENT_HOST an IP address, or is it just tcp://fluentbit:25888

Reply:
its an IP